### DISCUSSION TITLE
404 error after the "Setup directories and API endpoint" stage

### DISCUSSION THREAD

**Discussion by alex-wake** [Azure-Samples/graphrag-accelerator#52] [Q&A] [answered]
Hi! First time using Azure and trying to get the GraphRAG accelerator working. I've set my endpoint (the base/endpoint URL for the GraphRAG API (this is the Gateway URL found in the APIM resource)) but I can't get past the 404 error when trying to upload the wiki files. In the Portal I don't have any APIs or Products created in the APIM resource if that helps. Appreciate any guidance anyone can give me here. Thank you

**Comment by timothymeyers** [comment, marked as answer]
Hey @alex-wake this could be a lot of different things, so might need your help honing in on the root cause.

Can you share a sanitized copy of your deploy.sh output? Did that process result in SUCCESS?

Which notebook are you using to upload the files? (Quickstart or AdvancedQuickstart)?

Were you able to set your APIM OCP Subscription Key properly?
